docs/pages/machine-id/workload-identity.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Workload Identity
+description: Teleport's Workload Identity for SPIFFE
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. We are actively working on
+improving the feature and would love to [hear your feedback](mailto: product@goteleport.com).
+We currently do not recommend using this feature in production.
+</Admonition>
+
+Teleport's Workload Identity issues flexible short-lived identities intended
+for workloads. It is compatible with the industry-standard SPIFFE specification
+meaning that it can be used in place of other SPIFFE compatible identity
+providers.
+
+Teleport's Workload Identity brings a number of Teleport features you are
+already familiar with, such as:
+
+- Rich auditing of the issuance of SPIFFE SVIDs.
+- Powerful RBAC for limiting access to SPIFFE IDs to specific machines and
```

Comment:
I would remove qualifiers like "rich" and "powerful" that are subject to interpretation.

Reply:
I've made similar suggestions on docs reviews recently. Is there a section of the style guide I can link to for the next time this comes up?

Reply:
This is the closest thing to it: https://github.com/gravitational/docs/blob/main/docs-contributors/style-guide.md#voice

Reply:
2285838b13